src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.Expander.cs

```diff
@@ -764,9 +767,9 @@ private bool IsInvocationWithDynamicArguments(SimpleNameSyntax originalSimpleNam
                     {
                         foreach (var argument in invocationExpression.ArgumentList.Arguments)
                         {
-                            if (argument != null && argument.Expression != null)
+                            if (argument != null)
                             {
-                                var typeinfo = semanticModel.GetTypeInfo(argument.Expression);
+                                var typeinfo = semanticModel.GetTypeInfo(argument.Expression != null ? argument.Expression : argument.Declaration.Type);
```

Comment:
sure

---
In reply to: [68113256](https://github.com/dotnet/roslyn/pull/12159#discussion_r68113256) [](ancestors = 68113256)